components/closebutton/index.css

```diff
@@ -103,20 +103,9 @@ governing permissions and limitations under the License.
     --highcontrast-closebutton-background-color-default: ButtonFace;
     --highcontrast-closebutton-focus-indicator-color: ButtonText;
 
-    &:focus-ring {
+    &:focus-visible {
       &:after {
         forced-color-adjust: none;
-        box-shadow:
```

Comment:
Can you add some notes on why this was removed for me?

Reply:
Sure! I'll put this in the PR description too but this was all a duplication of the styles that are already applied to the `:after` element on lines 159-170. The ones left were the only ones unique to High Contrast mode, which is where this code is being applied.